publisher/publisher.go

```diff
@@ -80,6 +83,18 @@ func (h *HoneycombPublisher) Write(chunk string) {
 						"error": err,
 					}).Error("Unexpected error adding data to libhoney event")
 				}
+
+				// periodically provide updates to indicate work is actually being done
+				if time.Since(h.lastUpdateTime) >= time.Minute {
```

Comment:
I think this means that if rdslogs stops, so will the log messages, requiring the reader to infer from the _absence_ of a message that something is wrong (rather than seeing a message that says "zero messages". Is that right? Mildly :sadpanda: if it is; I prefer messages affirming that something isn't working. 

Also, any chance the interval could be configurable, taking a page out of honeytail's https://github.com/honeycombio/honeytail/blob/master/main.go#L63 flag?